BUILDING.md

```diff
@@ -300,53 +300,55 @@ To install this version of Node.js into a system directory:
 
 To verify the build:
 
-```console
-$ make test-only
+```text
+make test-only
 ```
 
 At this point, you are ready to make code changes and re-run the tests.
 
-If you are running tests before submitting a pull request, the recommended
-command is:
+If you are running tests before submitting a pull request, use:
 
-```console
-$ make -j4 test
+```text
+make test
 ```
 
 `make -j4 test` does a full check on the codebase, including running linters and
 documentation tests.
 
-Make sure the linter does not report any issues and that all tests pass. Please
-do not submit patches that fail either check.
-
-If you want to run the linter without running tests, use
+To run the linter without running tests, use
 `make lint`/`vcbuild lint`. It will lint JavaScript, C++, and Markdown files.
 
 If you are updating tests and want to run tests in a single test file
 (e.g. `test/parallel/test-stream2-transform.js`):
 
 ```text
```

Comment:
The command flags are inconsistent and have lots of highlighting surprises so I'd rather leave changing them as out of scope. In this case in particular, since there's a command prompt, `bash` might highlight it in incorrect ways.